fixtures/node_annotate.py

```diff
@@ -46,6 +49,8 @@ def pytest_configure(config):
 def pytest_addoption(parser):
     group = parser.getgroup('cfme')
     group.addoption('--tier', type=int, action='append', help='only run tests of the given tiers')
+    group.addoption('--requirement', type=str, action='append',
+        help='only run tests of the given requirement')
```

Comment:
Just a minor thing here to avoid possible confusion - `only run tests of the given requirements`.